### PR TITLE
add doc for serving media files from s3

### LINKED ISSUE #166: How can I serve (s3) media files in a HIPAA-compliant way?
https://aptible.zendesk.com/agent/tickets/2827

### PR DESCRIPTION
**Pull request by wcpines**
@fancyremarker for review.  Will fix https://github.com/aptible/support/issues/166

### CODE REVIEW

**Comment by wcpines** [comment]
@fancyremarker Second pair of eyes to catch any other typos would be :ok_hand:

**Comment by wcpines** [comment]
Added two sections, one to db:tunnel article, and one to database migrations article.

File: data/topics.yml [hunk 1]
```diff
@@ -96,6 +96,8 @@
       url: "paas/how-to-order-certs"
     - title: "How does the MAINTENANCE_PAGE_URL config setting work?"
       url: "paas/how-does-the-maintenance-page-url-config-setting-work"
+    - title: "How do I serve serve media files from Amazon S3?"
```
Comment (fancyremarker) [inline data/topics.yml]: "serve serve" –> "serve"

File: source/topics/paas/how-to-automate-database-migrations.md [hunk 2]
```diff
@@ -7,4 +7,25 @@ For example, to automate Rails migrations, you might have the following `.aptibl
 ```yaml
 before_release:
   bundle exec rake db:migrate
-```
\ No newline at end of file
+```
+
+If you want to deploy the same repo to different environments that require different rake tasks, we suggest referencing a bash script:
```
Comment (fancyremarker) [inline source/topics/paas/how-to-automate-database-migrations.md]: Pedantic: "bash" –> "Bash" (generally, refer to official [docs](https://www.gnu.org/software/bash/) as a spelling/capitalization guide.

File: source/topics/paas/how-to-serve-s3-media-files.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+This article covers _storing_ and _serving_ media files from S3 in a HIPAA-compliant manner
```
Comment (fancyremarker) [inline source/topics/paas/how-to-serve-s3-media-files.md]: Punctuation.

**Comment by fancyremarker** [comment]
Looks good. Minor fixes. Feel free to correct and merge without further review.

File: source/topics/paas/how-to-serve-s3-media-files.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+2. Client-side encryption (CSE, documented [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingClientSideEncryption.html)): This is the most challenging approach, but also gives you full control. You pick an encryption library, and implement the encryption/decryption logic.
+3. Server-side encryption with Amazon-provided keys (SSE, documented [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html)): This is the easiest approach, but the least secure. You need only specify that encryption should occur on PUT, and you never need to keep track of encryption keys. The downside is that if any of your privileged AWS accounts (or access keys) are compromised, all of your S3 data may be compromised and unprotected by a secondary key.
+
+
+There are two ways to serve S3 media files:
+
+1. Generate a pre-signed URL so that the client can access them directly from S3.
+2. Route all media requests through your app, fetch the S3 file within your app code, then re-serve it to the client.
+
+The first of these options is superior from a performance perspective.
```
Comment (rwjblue) [inline source/topics/paas/how-to-serve-s3-media-files.md:15]: I understand that this refers to the list directly above, but when I initially read it I thought it was saying that the first option in the encryption implementation style list was best.

**Pull request merged**